command/kv_metadata_get.go

```diff
@@ -117,6 +117,8 @@ func (c *KVMetadataGetCommand) Run(args []string) int {
 
 	delete(secret.Data, "versions")
 
+	outputPath(c.UI, path, true)
```

Comment:
Rather than passing a boolean here, I think it would be easier to read with a string:

```go
func outputPath(ui cli.Ui, path string, title string) {
      // ...
}

outputPath(c.UI, path, "Metadata Path")
```

Alternatively, it's also OK to have two functions, `outputSecretPath` and `outputMetadataPath`. I think I would lean towards just passing a string.